src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Field.cs

```diff
@@ -61,7 +61,7 @@ private BoundExpression RewriteTupleFieldAccess(BoundFieldAccess node, BoundExpr
 
             // make a field access for the most local access
             WellKnownMember wellKnownTypeMember = TupleTypeSymbol.GetTupleTypeMember(currentLinkType.Arity, fieldRemainder);
-            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(_compilation, wellKnownTypeMember, _diagnostics, syntax: node.Syntax);
+            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(_compilation, wellKnownTypeMember, _diagnostics, type: currentLinkType.OriginalDefinition, syntax: node.Syntax);
```

Comment:
I don't think there is an observable change yet. This will only come into play once we load TupleTypeSymbols from metadata, which we'll be working on next.
But from design discussion we agreed the lookup should be relative.